src/globalInstance.ts

```diff
@@ -15,10 +16,19 @@ export function getGlobalInstance(): Monitor {
 }
 
 export const monitored: Monitor['monitored'] = (...args) => getGlobalInstance().monitored(...args);
+
 /**
  * @deprecated since version 2.0
  */
 export const getStatsdClient: Monitor['getStatsdClient'] = (...args) => getGlobalInstance().getStatsdClient(...args);
 export const increment: Monitor['increment'] = (...args) => getGlobalInstance().increment(...args);
 export const gauge: Monitor['gauge'] = (...args) => getGlobalInstance().gauge(...args);
 export const timing: Monitor['timing'] = (...args) => getGlobalInstance().timing(...args);
+
+export const monitorMethod = <T>(prefix?: string, options: MonitoredOptions<T> = {}) => {
```

Comment:
I'd replace `monitorMethod` with a `function` (instead of const) and provide an overload which does not take the prefix parameter.

I want to avoid `@monitorMethod(undefined, {...})`.

Or, just make `prefix` required. What use cases do we have that we don't add some prefix to your monitor call?